test/config/interop-jd-translator/proxy-config.toml

```diff
@@ -4,7 +4,7 @@ upstream_port = 34265
 upstream_authority_pubkey = "3VANfft6ei6jQq1At7d8nmiZzVhBFS4CiQujdgim1ign"
 
 # Local Mining Device Downstream Connection
-downstream_address = "0.0.0.0"
+downstream_address = "127.0.0.1"
```

Comment:
why this?

Reply:
Only way it works on Windows afaik.

Reply:
no, not really important.
I think that if you are making it work with a miner in your LAN (and not just a cpu miner), than you have to set it back to 0.0.0.0. I am not sure anyway

Reply:
> why this?

I changed it because the MG test was failing, but it seems that this is not the fix

Reply:
it shouldn't be that the problem!

Reply:
LGTM anyway